src/cmd/build.js

```diff
@@ -59,14 +60,23 @@ export async function getDefaultLocalizedName(
 ): Promise<string> {
 
   let messageData: LocalizedMessageData;
+  let messageContents: string | Buffer;
   let extensionName: string = manifestData.name;
 
   try {
-    messageData = JSON.parse(await fs.readFile(messageFile));
+    messageContents = await fs.readFile(messageFile);
   } catch (error) {
     throw new UsageError(
-      `Error reading or parsing file ${messageFile}: ${error}`);
+      `Could not read messages.json file at ${messageFile}: ${error}`);
   }
+
+  try {
+    messageData = parseJSON(messageContents, messageFile);
+  } catch (error) {
+    throw new UsageError(
+      `Error parsing manifest.json at ${messageFile}: ${error}`);
```

Comment:
@kumar303 do we want to create an explicit subclass of UsageError for this error scenario? (e.g. InvalidMessages)